packages/cli/cloud/src/logout/action.ts

```diff
@@ -11,9 +13,26 @@ export default async (ctx: CLIContext) => {
     return;
   }
   const cloudApiService = await cloudApiFactory(token);
+  const config = await cloudApiService.config();
+  const cliConfig = config.data;
+
   try {
-    // we might want also to perform extra actions like logging out from the auth0 tenant
     await eraseToken();
+
+    openModule.then((open) => {
+      open
+        .default(
+          `${cliConfig.baseUrl}oidc/logout?client_id=${encodeURIComponent(
+            cliConfig.clientId
+          )}&logout_hint=${encodeURIComponent(token)}
```

Comment:
Do you have a preview of the logout page?

Reply:
<img width="927" alt="7a95c833-d2d8-4060-b5b4-35d24b2f911d" src="https://github.com/strapi/strapi/assets/55534657/9b86a52d-3d1a-471f-add7-efcaae5d4c96">
Yes, there you go! Will add it to the description